features/features.go

```diff
@@ -139,6 +143,8 @@ var featuresExported = map[SnapdFeature]bool{
 
 	RefreshAppAwarenessUX: true,
 	AspectsConfiguration:  true,
+
+	AppArmorPrompting: true,
```

Comment:
Why does this need to be exported?

Reply:
Also if it's necessary then we need to add it to the control file test as well

Reply:
I believe it is necessary because we check if prompting is enabled here: https://github.com/snapcore/snapd/pull/12964/files#diff-1bc7f26dd750993e7bf7bb0c8feb2e6d664f99313b5069f1dd68f4c3a66d51f9R19

Reply:
I suppose we could use `features.Flag()` here instead. I'm not really sure of the benefits of `IsEnabled()` vs `Flag()`, other than that the former can work on non-exported functions.

More generally though, we want users to know whether apparmor prompting is enabled (as well as other experimental features), and we'll expose this via the `/v2/system-info` endpoint. To do this, does `AppArmorPrompting` need to be exported?

Reply:
exported is used if snapd own tools outside of the main snapd daemon needs to know about the status of the feature. Does for example snap-confine need changes based on the feature?